python3: can't open file '/mnt/c/AItools/segment-anything/run_batch_extraction.py': [Errno 2] No such file or directory
